midealocal/cloud.py

```diff
@@ -178,10 +178,10 @@ async def list_appliances(
     ) -> dict[int, dict[str, Any]] | None:
         raise NotImplementedError()
 
-    async def get_device_info(self, device_id: str) -> dict[str, Any] | None:

```

Comment:
Please check with #37 if all is fine

Reply:
The issue was on test, I was unable to commit because I was passing an int to that method.

Inside the method a cast to int was made and on every usage a cast to string as made.

Reply:
Ok, didn't check myself; just asked for a double check from you ;-)